my-dev-portal/src/components/navigation/mobile/mobile-nav-bar-buttons.js

```diff
@@ -1,15 +1,33 @@
-import { useAuth0 } from "@auth0/auth0-react";
 import React from "react";
+import { useOktaAuth } from "@okta/okta-react";
+import { useAuth0 } from "@auth0/auth0-react";
 import { LoginButton } from "../../buttons/login-button";
 import { LogoutButton } from "../../buttons/logout-button";
 import { SignupButton } from "../../buttons/signup-button";
 
 export const MobileNavBarButtons = () => {
-  const { isAuthenticated } = useAuth0();
+
+  const { authState } = useOktaAuth(); 
+  const oktaIsAuthenticated = authState?.isAuthenticated;
+
+  const { isAuthenticated: auth0IsAuthenticated } = useAuth0();
+
+  let isAuthenticated;
+
+  if(process.env.REACT_APP_AUTH_PROVIDER === "Okta") {
+    isAuthenticated = oktaIsAuthenticated;
+  } else if(process.env.REACT_APP_AUTH_PROVIDER === "Auth0") {
+    isAuthenticated = auth0IsAuthenticated;
+  }
 
```

Comment:
I see this section of code is used a seveal times. maybe consolidate? but if not easy. do not worry about it.

Reply:
agreed. we should add to refactor list.